### PR TITLE
Add info regarding creating new repos

### PR DESCRIPTION
**Pull request by henning-roos**
### Applicable Issues
None

### Description of the Change
In order to promote similar process and handling across the community a template repository was created to achieve this. This PR will make this visible in the contributing guidelines.

### Benefits
In order to promote similar process and handling across the community a template repository was created to achieve this. This PR will make this visible in the contributing guidelines.

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Henning Roos  henning.roos@ericsson.com

### CODE REVIEW

**Comment by henning-roos** [comment]
I need some more reviews on this PR...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,3 +81,6 @@ When John Doe makes a subsequent contribution the following year, the notice wil
 
 > Copyright 2018-2019 Ericsson AB and others.  
 > For a full list of individual contributors, please see the commit history.
+
+## How to Create a New Eiffel Repository
+In order to promote similar process and handling across the community, new Eiffel repositories are created by coping the contents of the eiffel-repository-template (https://github.com/eiffel-community/eiffel-repository-template), then following the guidelines given in that repository.
```
Comment (m-linner-ericsson) [inline CONTRIBUTING.md]:
```suggestion
In order to promote a similar process and handling across the community, new Eiffel repositories should be created by copying the contents of the eiffel-repository-template (https://github.com/eiffel-community/eiffel-repository-template), then follow the guidelines given in that repository.
```

**Review by m-linner-ericsson — COMMENTED** [review]
The instruction is in passive voice but that maybe is not an issue?

**Review by m-linner-ericsson — APPROVED** [review]
Looks good

**Pull request merged**